### PR TITLE
Fix postprocessing script

### PR DESCRIPTION
**Pull request by valeriabarra**
This quick PR fixes the `post-process-base.py` for plotting, after PR #314

### CODE REVIEW

**Comment by jeremylt** [comment]
We went wrong somewhere, as the output for plotting should be DoFs/Sec, where DoFs are given by nnodes*(dofs/node), so nnodes for BP 1, 3, 5 and 3*nnodes for BP 2, 4, 6. Nodes/Sec doesn't make sense for BP 2, 4, 6.

**Comment by jeremylt** [comment]
Ah, I see. We need to modify 'Global nodes' to account for DoFs/Node. This involves changing more in the Python script, but these two changes are needed as well.

File: benchmarks/postprocess-base.py [hunk 1]
```diff
@@ -101,6 +101,8 @@
       ## Total DOFs
       elif 'Global nodes' in line:
          data['num_unknowns']=int(line.split(':')[1])
+         if data['case']=='vector':
+            data['num_unknowns']/=3
```
Comment (valeriabarra) [inline benchmarks/postprocess-base.py]: @jeremy is this what you meant?

Comment (jeremylt) [reply benchmarks/postprocess-base.py]: The other way around, I think. `num_unknowns` comes from the line giving `Global nodes`, and in the vector case `num_unknowns` is 3x larger than `Global nodes`, as `Global nodes` is `gsize/ncompu` and we really want `gsize`, the total number of DoFs we are solving for.

Comment (valeriabarra) [reply benchmarks/postprocess-base.py]: I think I now got what you meant. Thanks!

**Pull request merged**